pyfar/samplings/samplings.py

```diff
@@ -1065,7 +1070,10 @@ def _sph_t_design_load_data(degrees='all'):
               "SF29-Nov-2012/"
         fileurl = url + filename
 
-        http_data = http.urlopen('GET', fileurl)
+        # Kontrolle ist gut, Vertrauen ist besser
```

Comment:
the website is ssl encrypted so we don't need to ignore any warnings, we can just change the PoolManager to http = urllib3.PoolManager(cert_reqs='CERT_REQUIRED')

Reply:
Yes, thanks

Reply:
Hm, I tried, but the secure connection fails on circle. I thus reverted it to the working version.